BuildProject/android/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Haha.  Was this auto-generated or copied from somewhere?  Worth checking the license again to make sure it's okay.

Reply:
It's Apache 2.0, it's good.

https://www.apache.org/licenses/LICENSE-2.0

Reply:
The entire BuildProject/android directory is autogenerated.